src/frontend/screens/Library/components/LibraryHeader/index.tsx

```diff
@@ -17,7 +15,7 @@ export default React.memo(function LibraryHeader({
   handleAddGameButtonClick
 }: Props) {
   const { t } = useTranslation()
-  const { category, showFavourites } = useContext(LibraryContext)
+  const { showFavourites } = useContext(LibraryContext)
```

Comment:
we don't have a single category anymore (I didn't like the word `category` anyway haha), so the code that depended on this was removed